canopy_main.ml

```diff
@@ -41,10 +41,10 @@ module Main  (C: CONSOLE) (S: STACKV4) (RES: Resolver_lwt.S) (CON: Conduit_mirag
     let module Store = Canopy_store.Store(C)(Context)(Inflator) in
     let open Canopy_utils in
     let cache = ref (KeyMap.empty) in
-    let open Canopy_config in
-    let config = config () in
+    Store.pull console >>= fun () ->
+    Store.fill_cache cache >>= fun _ ->
```

Comment:
We can only error inside Canopy_syndic.atom, a few lines further down, if I am right...
So we could catch the error in that function and return an empty list for the atom feed.

An alternative to raising an exception would be to generate an uuid (if it's missing) with the uuidm library (https://github.com/dbuenzli/uuidm). I think it is pure OCaml...